src/VirtualClient/VirtualClient.Actions/SysbenchOLTP/SysbenchOLTPServerExecutor.cs

```diff
@@ -100,15 +110,18 @@ await process.WaitForExitAsync(cancellationToken)
             });
         }
 
-        private async Task ConfigureMySQLPrivileges(CancellationToken cancellationToken)
+        private async Task ConfigureMySQLPrivilegesAsync(CancellationToken cancellationToken)
         {
             string workingDirectory = this.GetPackagePath(this.PackageName);
 
+            string dropUserCommand = $"mysql --execute=\"DROP USER 'sbtest'@'{this.ClientIpAddress}'\"";
```

Comment:
Where did the "sbtest" user name come from? If it's from the dependency class, make a parameter on both dependency and action

Reply:
oooooh! okay i wasn't sure if i was able to retrieve client/server ip addresses from the mysql dependency directly. let me reconfigure all this then